go.mod

```diff
@@ -4,12 +4,12 @@ go 1.12
 
 require (
 	cloud.google.com/go v0.41.0 // indirect
-	github.com/Azure/azure-sdk-for-go v32.6.0+incompatible
+	github.com/Azure/azure-sdk-for-go v32.4.0+incompatible
```

Comment:
please rebase and back out changes to this file

Reply:
done